thoth/slo_reporter/sli_apis/sli_user_api.py

```diff
@@ -87,7 +90,7 @@ def _query_sli(self) -> List[str]:
             "avg_up_time": f"avg_over_time(up{query_labels_up}[{self.configuration.interval}])",
         }
 
-    def _evaluate_sli(self, sli: Dict[str, Any]) -> Dict[str, float]:
+    def _evaluate_sli(self, sli: Dict[str, Any], last_week_data) -> Dict[str, float]:
```

Comment:
Step 7. Hope to get that data, for now it's just `str`, but I think, it should be easy to convert to `pd.Dataframe`. But, do you think converting to `pd.Dataframe` is the right choice? Just let me show you example for `user_api-2020-10-01.csv` below:
`2020-10-01,1601573293858,99.622,100.0`
My only concern is they don't have column names and that's why I' stuck how to use this data to calculate the difference.

Reply:
> Step 7. Hope to get that data, for now it's just `str`, but I think, it should be easy to convert to `pd.Dataframe`. But, do you think converting to `pd.Dataframe` is the right choice? Just let me show you example for `user_api-2020-10-01.csv` below:
> `2020-10-01,1601573293858,99.622,100.0`
> My only concern is they don't have column names and that's why I' stuck how to use this data to calculate the difference.
Fair point, you can take the column names based on the data for that metric_class in html_inputs created with `_evaluate_sli`:
 
for each CSV you always have: defaults = ["datetime", "timestamp"] and metric_class_columns = "avg_percentage_successfull_request", "avg_up_time"] (for example for user-API), we need to generalize it a bit later as there every context is very specific at the moment. But this should be the idea